internal/sftp/ssh.go

```diff
@@ -50,12 +51,14 @@ func sshConnect(cfg Config) (*ssh.Client, error) {
 		},
 		HostKeyCallback: hostcallback,
 		Timeout:         5 * time.Second,
+		User:            cfg.User,
 	}
 
 	// Connect to the server.
 	address := net.JoinHostPort(cfg.Host, cfg.Port)
 	conn, err := ssh.Dial("tcp", address, sshConfig)
 	if err != nil {
+		logger.Error(err, "SSH dial:", "address", address, "user", cfg.User)
```

Comment:
One issue with logging an error here is that we may also be doing the same above in the stack. That's extra work for the system administrator if we're producing two errors for the same event. An alternative could be to use `logger.Info("...", "err", err)` or `logger.V(2).Info("...", "err", err)` to avoid the pollution.